ui/lib/core/addon/components/text-file.hbs

```diff
@@ -18,75 +18,40 @@
       </label>
     </div>
     <div class="level-right">
-      <div class="control is-flex">
-        <Input
-          data-test-text-toggle
-          id="use-text-{{this.elementId}}"
-          class="toggle is-success is-small"
-          @type="checkbox"
-          @checked={{this.showTextArea}}
-          {{on "change" (fn (mut this.showTextArea) (not this.showTextArea))}}
-        />
-        <label for="use-text-{{this.elementId}}" class="has-text-weight-bold is-size-8">
-          Enter as text
-        </label>
-      </div>
+      <Toggle
```

Comment:
The new HDS buttons looked awkward with the boxier input styling so updated this file quite a bit
![text-file](https://github.com/hashicorp/vault/assets/68122737/52a397c8-2a2f-42f3-a238-fed1970ba5b2)

Reply:
It looks like the eye icon overlaps the scrollbar but it looks like that is from HDS within the masked input component?